src/app/components/forms/post/post.ts

```diff
@@ -440,14 +425,16 @@ export default class FormPost extends BaseForm<FormPostModel>
 			this.accessPermissionsEnabled = true;
 		}
 
-		if (model.hasArticle) {
+		if (model.has_article) {
 			this.longEnabled = true;
 		}
 
 		await this.fetchTimezones();
 	}
 
 	onLoad(payload: any) {
+		// Pull any post information that may not already be loaded in.
+		this.formModel.assign(payload.post);
```

Comment:
This breaks creating posts since it sets the status to temp, which gets sent back during save.
What data exactly do we want to fetch here?

Reply:
specifically populating article_content works.

Reply:
Dang. It's mainly for article_content, but I didn't want it to be just for that. I wanted it generic to essentially make sure the post is fully loaded from server before you can work on it. Just in case there's more lazy loaded fields.